tests/azure/test_functional_cloudinit.py

```diff
@@ -955,10 +955,9 @@ def _verify_authorizedkeysfile(self, keyfiles):
         # 4. Verify can login and no unexpected files in ~/.ssh
         self.assertTrue(self.session.connect(timeout=10),
                         "Fail to login after run ssh module")
-        files = self.session.cmd_output(
-            "find /home/{}/.ssh/*|grep -vE '(id_rsa|known_hosts)'".format(self.vm.vm_username))
-        self.assertEqual(len(files.split()), 1,
-                         "There are unexpected files under ~/.ssh: {}".format(files))
```

Comment:
Is there any way to also keep this checkpoint?(Ensure that no other files in the folder expect keyfiles and known_hosts)

Reply:
Revised as suggested. Thanks.